extensions/ql-vscode/src/view/common/DataGrid.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from "react";
+import { forwardRef } from "react";
+import { styled } from "styled-components";
+
+/*
+ * A drop-in replacement for the VSCodeDataGrid family of components.
+ *
+ * The difference is that the `display: grid` styling is applied to `DataGrid`, whereas
+ * in the VS Code version that styling is applied to `VSCodeDataGridRow`. This gives
+ * column alignment across rows in situation with dynamic contents. It also allows
+ * for cells to span multiple rows and all the other features of data grids.
+ */
+
+const StyledDataGrid = styled.div<{ $gridTemplateColumns: string | number }>`
```

Comment:
Oops, I agree this isn't useful for the `gridTemplateColumns` property.

On `gridColumn` it lets you write `gridColumn={5}` instead of `gridColumn={"5"}`, but I must have accidentally copied the type to here as well.